crates/noirc_evaluator/src/ssa_refactor/ir/cfg.rs

```diff
@@ -127,14 +127,16 @@ impl ControlFlowGraph {
 
 #[cfg(test)]
 mod tests {
-    use crate::ssa_refactor::ir::{instruction::TerminatorInstruction, map::Id, types::Type};
+    use crate::ssa_refactor::ir::{
+        function::RuntimeType, instruction::TerminatorInstruction, map::Id, types::Type,
+    };
 
     use super::{super::function::Function, ControlFlowGraph};
 
     #[test]
     fn empty() {
         let func_id = Id::test_new(0);
-        let mut func = Function::new("func".into(), func_id);
+        let mut func = Function::new("func".into(), func_id, RuntimeType::Normal);
```

Comment:
Can we have RuntimeType be an attribute that is added later? This would allow testing code like this to ignore it and would allow us to add other attributes as needed later.

Reply:
I mean removing it from the constructor here and adding a mutator method so it can be optionally set later.

Reply:
Done